### PR TITLE
Allow pasting in debug window #100235

### LINKED ISSUE #100235: Allow right-click paste inside the debug console
image

- VSCode Version: 1.46

Steps to Reproduce:

1. Open the debug debug console
2. Right-click inside the console view

There should be an option to paste text. Currently, pasting is only allowed when right-clicking the very narrow text input area. 

The debug console should be more similar to the terminal, which allows for right-click pasting anywhere. 

Does this issue occur when all extensions are disabled?: Yes/No

### PR DESCRIPTION
**Pull request by daniel-j-davis**
This PR fixes #100235 

As discussed in the issue, this adds paste functionality to the debug window so that you can paste anywhere, instead of restricting it to the input field.

Here's a gif that shows how this functionality works - https://i.gyazo.com/f1cfe66ae6ad3ec7718c308c1c702d11.gif

### CODE REVIEW

**Comment by isidorn** [comment]
@daniel-j-davis great work! Love it!
I left two comments in code which when addressed I can merge this in

File: src/vs/workbench/contrib/debug/browser/repl.ts [hunk 1]
```diff
@@ -611,6 +611,13 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
 			this.replInput.focus();
 			return Promise.resolve();
 		}));
+		actions.push(new Action('debug.replPaste', localize('paste', 'Paste'), undefined, true, async () => {
```
Comment (isidorn) [inline src/vs/workbench/contrib/debug/browser/repl.ts:614]: Please use "Paste" not 'Paste', user facing string we double quote.

File: src/vs/workbench/contrib/debug/browser/repl.ts [hunk 2]
```diff
@@ -611,6 +611,13 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
...
+			const clipboardText = await this.clipboardService.readText();
```
Comment (isidorn) [inline src/vs/workbench/contrib/debug/browser/repl.ts:615]: Make sure this action is before the `Collapse All` action.
So it should be Copy, Copy All, Paste, Collapse All

File: src/vs/workbench/contrib/debug/browser/repl.ts [hunk 3]
```diff
@@ -611,6 +611,13 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
...
+			if (clipboardText) {
+				this.replInput.setValue(this.replInput.getValue().concat(clipboardText));
+			}
+			return Promise.resolve();
```
Comment (isidorn) [inline src/vs/workbench/contrib/debug/browser/repl.ts:619]: No need to return anything, this line is not needed. `async` magic will do all for you.

**Review by isidorn — CHANGES_REQUESTED** [review]

**Comment by daniel-j-davis** [comment]
Thanks for the feedback, @isidorn. I've made the changes that you requested in the latest two commits!

**Comment by isidorn** [comment]
Great work, merging in.

**Pull request merged**